### ISSUE TITLE
Questions about the refSR on dual-camera data

### ISSUE THREAD

**Issue by JQJ001** [THU-luvision/CrossNetPP#1]
Hello, this is an amazing job! Thanks for sharing.

I have several questions to ask you:
1) The  link for download pretrained_models is not working. 
1) How to train on the dual-camera dataset after loading the pretrained model on light-field dataset Flower? What's the loss function? You said "warping loss is disabled" and "there does not exist real ground-truth for dual-camera data" in the paper. 
3) I wonder if you have any plans to release the hikvision dual-camera dataset?

Thank you very much. Looking forward to your reply.
